docs/sources/tempo/setup/operator/monolithic.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Monolithic deployment
+description: Set up a Tempo deployment in monolithic mode
+menuTitle: Monolithic deployment
+weight: 500
+aliases:
+- /docs/tempo/operator/monolithic
+---
+
+# Monolithic deployment
+
+The `TempoMonolithic` Custom Resource (CR) creates a Tempo deployment in [Monolithic mode](/docs/tempo/latest/setup/deployment/#monolithic-mode).
+In this mode, all components of the Tempo deployment (compactor, distributor, ingester, querier and query-frontend) are contained in a single container.
+
+This type of deployment is ideal for small deployments, demo and test setups, and supports storing traces in memory, in a Persistent Volume and in object storage.
+
+**Note:** The monolithic deployment of Tempo does not scale horizontally. If you require horizontal scaling, please use the `TempoStack` CR for a Tempo deployment in [Microservices mode](/docs/tempo/latest/setup/deployment/#microservices-mode).
+
```

Comment:
You can use the [admonition shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition) to format these notes. 

We also changed how we're doing links recently. While the links you have will work, you can also now use the full URL, just like you would for regular Markdown: `[link text](http://grafana.com/stuff/here]`. When a link goes to a versioned product like Tempo, we're replacing /latest with something like <TEMPO_VERSION>. This allows the URL to be adapted if a page is backported for a previous version, for example. The link would go to the correct version of that page instead of to /latest/. For more information: https://grafana.com/docs/writers-toolkit/write/links/

```suggestion
{{< admonition type="note" >}}
The monolithic deployment of Tempo does not scale horizontally. If you require horizontal scaling, please use the `TempoStack` CR for a Tempo deployment in [Microservices mode](https://grafana.com/docs/tempo/<TEMPO_VERSION>/setup/deployment/#microservices-mode).
{{< /admonition >}}
```

Reply:
Thank you, I'll use that format next time.